--- lib/python3.5/site-packages/fastparquet/writer.py	2017-03-23 13:31:43.000000000 -0500
+++ lib/python3.5/site-packages/fastparquet/writer.py.new	2017-03-23 13:34:30.000000000 -0500
@@ -620,7 +620,7 @@
                                           parquet_thrift.Type.INT96]
         elif has_nulls is not True and has_nulls is not False:
             #col_has_nulls = column in has_nulls
-	    pass
+            pass
         if col_has_nulls and data[column].dtype.kind != 'i':
             se.repetition_type = parquet_thrift.FieldRepetitionType.OPTIONAL
         fmd.schema.append(se)
